scripts/qa_crowbarsetup.sh

```diff
@@ -2888,6 +2888,11 @@ function deploy_single_proposal()
 {
     local proposal=$1
 
+    # skip a proposal
+    if [ -n "skip_proposal_$proposal" ]; then
```

Comment:
this will always be true. You maybe want something like
eval "echo \$skip_proposal_$proposal"